brainstormforce/astra
# refactor: Partial refresh calls to the backend.

### Description
- While doing partial refresh it was recalculation static array which is the same always. 

### Screenshots
- NA

### Types of changes
- Created a database option with autoloading false.
- While doing the partial refresh, fetching the database and reset the existing array instead of recalculating.
- Delete cached data on publishing.

### How has this been tested?
- Check the possibility of breaking the application.
- Check partial /. page refresh.

### Checklist:
